src/lib/pages.ts

```diff
@@ -102,7 +102,9 @@ export class Pages {
     public canvas: Page,
     public canvasWidth: number,
     public canvasHeight: number,
-    public updateState: () => void
+    public updateState: () => void,
+    // TODO: this should be handled by the synchronization class and not this class
```

Comment:
no? i think this is fine

Reply:
When collaboration is added

Reply:
idk man, this is why we have issues, this should really go to the issue comments and not here. i think i've generally avoided putting TODOs in the code for this reason